docs/pages/jpsreport/jpsreport_profiles.md

```diff
@@ -0,0 +1,38 @@
+---
+title:  Density profiles
+keywords: analysis
+tags: [jpsreport, jpsreport_tutorial]
+sidebar: jupedsim_sidebar
+folder: jpsreport
+summary: This page shows how to get profiles of density, velocity and specific flow.
+permalink: jpsreport_profiles.html
+last_updated: Dec 21, 2019
+---
+
+{%include note.html content="this tutorial needs links to data"%}
+ 
+Here is an example extracted from a T-Junction experiment: 
+
+![T-junction]({{ site.baseurl }}/images/Figue4-4-3.png)
```

Comment:
This image is not displayed in the preview.

Reply:
Good catch. I forgot to upload it. Done!